flexmeasures/data/scripts/clean_database.sh

```diff
@@ -48,11 +97,11 @@ then
   response=${response,,} # make lowercase
   if [[ "$response" =~ ^(yes|y)$ ]]; then
      delete_database $1
```

Comment:
Within that function (and then when we call it here) we should also check for success. I had a psql session open which was connected to the database I wanted to re-create, and then got this:

```
± make clean-db db_name=flexmeasures_test db_user=flexmeasures_test
./flexmeasures/data/scripts/clean_database.sh flexmeasures_test flexmeasures_test
[sudo] password for nicolas: 
flexmeasures_test exists
Make a backup first? [y/N] 
This will drop your database and re-create a clean one. Continue?[y/N] y
Deleting database ...
dropdb: error: database removal failed: ERROR:  database "flexmeasures_test" is being accessed by other users
DETAIL:  There is 1 other session using the database.
Creating a new database ...
createdb: error: database creation failed: ERROR:  database "flexmeasures_test" already exists
flexmeasures_test is already available.
Connect flexmeasures_test to flexmeasures_test 
GRANT
Grant required privileges
GRANT
Creating cube extension in flexmeasures_test ...
You are now connected to database "flexmeasures_test" as user "postgres".
ERROR:  extension "cube" already exists
Creating earthdistance extension in flexmeasures_test ...
You are now connected to database "flexmeasures_test" as user "postgres".
ERROR:  extension "earthdistance" already exists
Updating database structure ...
```

It should stop after the first error (dropping not possible).

Reply:
In another terminal:

```
sudo -i -u postgres psql
\connect flexmeasures_test
```

Now there is an active connection to the database (I use flexmeasures_test here, which I try to create with your script), and the script should complain like it did to me.